source/administration/batch-framework.rst

```diff
@@ -79,16 +79,42 @@ The :mc:`mc batch` commands include
 Access to ``mc batch``
 ----------------------
 
-You can use MinIO's :ref:`Policy Based Access Control <minio-policy>` and the :ref:`administrative policy actions <minio-policy-mc-admin-actions>` to restrict who can start a batch job, retrieve a list of running jobs, or describe a running job.
+The batch function itself is a server process not limited by access keys or policies.
```

Comment:
I think this means if you have shell access we can't stop you?

Perhaps something along these lines?
```suggestion
The batch function itself is a server process, with access control managed by OS permissions rather than MinIO access keys or policies.
```

Reply:
Tweaked the wording and expanded the section a little bit to hopefully better clarify my intent with this section.